v2/validator.go

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package v2
+
+import (
+	"github.com/go-playground/locales/en"
+	ut "github.com/go-playground/universal-translator"
+
+	"github.com/go-playground/validator/v10"
+)
+
+var (
+	val   *validator.Validate
+	trans ut.Translator
+)
+
+func NewValidator() *validator.Validate {
```

Comment:
What is this used for? It is very cryptic.

Reply:
And it's public, please add godoc...